src/plugins/dashboard/public/dashboard_container/embeddable/api/run_save_functions.tsx

```diff
@@ -261,6 +265,16 @@ export async function runInteractiveSave(this: DashboardContainer, interactionMo
         this.saveNotification$.next();
 
         resolve(saveResult);
+
+        if (redirectTo) {
+          redirectTo({
+            id: saveResult.id,
+            editMode: true,
+            useReplace: true,
+            destination: 'dashboard',
+          });
+        }
```

Comment:
I'm curious - why do it this way, rather than doing it the way we do for the duplicate button in the top nav? 🤔

Reply:
Got it thank you for zooming and clarifying what you meant! [1dca243](https://github.com/elastic/kibana/pull/189404/commits/1dca243c105cde139b7a510e3389c3d67bd28319) for changes